htdocs/theme/raw/sass/components/_list-group.scss

```diff
@@ -40,6 +40,10 @@
         border-left: 0;
         border-right: 0;
     }
+
+    &.attachment-toggle {
+        border-left: 1px solid #ddd;
```

Comment:
Needs an explanation at this point I think (just put a comment in to say why a toggle in a listgroup needs a left border :) )